packages/client-core/src/util/contextParams.ts

```diff
@@ -0,0 +1,40 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { LogParamsObject } from '@etherealengine/common/src/logger'
+
+/**
+ * @function clientContextParams
+ * @description This function will collect contextual parameters
+ * from url's query params
+ */
+export function clientContextParams(params: LogParamsObject) {
+  const urlParams = new URLSearchParams(window.location.search)
```

Comment:
why not the entire URL href?

Reply:
Yes, I agree we will need to have entire URL. Because subdomain will tell which account the location was running for.

Reply:
Well, I'm just trying to avoid the necessity to parse or to decode params down the road, if I don't do it I would be passing the responsibility ball to someone else, this time I'm passing the entire thing but as params which would be much more helpful for Balasz than the entire URL as a string.

Reply:
we should put these on a named object then, rather than polluting the namespace of the object returned

Reply:
and we should include the domain name and URL pathname too